docusaurus.config.js

```diff
@@ -39,18 +39,33 @@ module.exports = {
       {
         id: "optimize",
         path: "optimize",
+        includeCurrentVersion: false,
         routeBasePath: "optimize",
         beforeDefaultRemarkPlugins: [versionedLinks],
-        sidebarPath: require.resolve("./optimize_sidebars.js"),
+        sidebarPath: require.resolve(
```

Comment:
rather than moving the archived sidebars file into the current version location, I pointed the current version at the archived version's sidebars file.